### ISSUE TITLE
When "workbench.editor.showTabs" is false, the user should still be able to drag title to create second pane

### ISSUE THREAD

**Issue by Ivo-Evans** [microsoft/vscode#141736]
Once editor tabs are disabled, you can still create panes, for instance by dragging from the explorer or adding custom keybindings (I use `"workbench.action.editorLayoutTwoColumns"`). However you lose the ability to create a pane by dragging the title of the active editor to one side of the screen. I can see how this happened and it does make some sense. However I think it would be most intuitive if, when you dragged the current pane title to one side of the screen, it did the same as `"workbench.action.editorLayoutTwoColumns"` and the surrounding family of keybindings.

Basically users that don't use tabs still use panes and they should have an intuitive experience with them - panes don't rely on the possibility of having multiple tabs per pane.

**Comment by bpasero** [comment]
The editor can be dragged, but you are essentially dragging the entire stack of editors, not the top one.
